### PR TITLE
fix: fix bug where parameters where wrongly used

### PR DESCRIPTION
**Pull request by rahalan**

### CODE REVIEW

File: avm/res/compute/virtual-machine/main.bicep [hunk 1]
```diff
@@ -617,10 +617,10 @@ resource vm_autoShutdownConfiguration 'Microsoft.DevTestLab/schedules@2018-09-15
             notificationLocale: contains(autoShutdownConfig, 'notificationLocale')
               ? autoShutdownConfig.notificationLocale
               : 'en'
-            timeInMinutes: contains(autoShutdownConfig, 'notificationWebhookUrl')
+            webhookUrl: contains(autoShutdownConfig, 'notificationWebhookUrl')
```
Comment (eriqua) [inline avm/res/compute/virtual-machine/main.bicep:620]: If there a specific reason to renamr the original properties? If not I'd suggest to keep them all aligned. E.g., 
```suggestion
            webhookUrl: contains(autoShutdownConfig, 'webhookUrl')
```

File: avm/res/compute/virtual-machine/main.bicep [hunk 2]
```diff
@@ -617,10 +617,10 @@ resource vm_autoShutdownConfiguration 'Microsoft.DevTestLab/schedules@2018-09-15
...
               ? autoShutdownConfig.notificationWebhookUrl
```
Comment (eriqua) [inline avm/res/compute/virtual-machine/main.bicep:621]:
```suggestion
              ? autoShutdownConfig.webhookUrl
```

File: avm/res/compute/virtual-machine/main.bicep [hunk 3]
```diff
@@ -617,10 +617,10 @@ resource vm_autoShutdownConfiguration 'Microsoft.DevTestLab/schedules@2018-09-15
...
               : ''
-            webhookUrl: contains(autoShutdownConfig, 'notificationTimeInMinutes')
+            timeInMinutes: contains(autoShutdownConfig, 'notificationTimeInMinutes')
```
Comment (eriqua) [inline avm/res/compute/virtual-machine/main.bicep:623]: Same suggestion for all autoshutdown properties. It's preferable to keep them aligned to the original name

File: avm/res/compute/virtual-machine/main.bicep (hunk 1 above)
Comment (rahalan) [reply avm/res/compute/virtual-machine/main.bicep:620]: @eriqua it is a sub property of notifications, thus the parent property before

File: avm/res/compute/virtual-machine/main.bicep (hunk 2 above)
Comment (rahalan) [reply avm/res/compute/virtual-machine/main.bicep:621]: @eriqua it is a sub property of notifications, thus the parent property before

File: avm/res/compute/virtual-machine/main.bicep (hunk 3 above)
Comment (rahalan) [reply avm/res/compute/virtual-machine/main.bicep:623]: @eriqua it is a sub property of notifications, thus the parent property before

File: avm/res/compute/virtual-machine/main.bicep (hunk 1 above)
Comment (eriqua) [reply avm/res/compute/virtual-machine/main.bicep:620]: If we follow that as a rule then I guess it should also be dailyRecurrenceTime and dailyRecurrenceTimeZoneId? Probably an overkill but I'll leave the decision to you as a module owner. I'll approve and keep this open for you to resolve if you want to merge as is.

**Review by eriqua — DISMISSED** [review]
🆗

File: avm/res/compute/virtual-machine/main.bicep (hunk 1 above)
Comment (rahalan) [reply avm/res/compute/virtual-machine/main.bicep:620]: @eriqua changed to dailyRecurrenceTime. The TimeZone is not part of the recurrence. The main reason for using the parent in the first place was, that status was already used by the parent, thus I needed a new name for the notification status

**Review by eriqua — APPROVED** [review]
🆗

**Pull request merged**